include/oneapi/dpl/experimental/kt/esimd_radix_sort_cooperative.h

```diff
@@ -201,7 +207,7 @@ void cooperative_kernel(sycl::nd_item<1> idx, size_t n, const InputT& input, Key
             // global sync()
             // each tg read global scan buffer, add with prev tg, then put to slm incoming buffer
             if (local_tid == 0) {
-                global_sync(p_sync_buffer, stage*4+0, tg_count, tg_id, local_tid);
+                global_sync(p_sync_buffer, stage*SYNC_BUFFER_PER_STAGE+0, tg_count, tg_id, local_tid);
```

Comment:
As far as I can see, `local_tid` is always 0 when `global_sync` is called, and moreover the corresponding parameter is not used by the function. If it is not needed, let's remove.